packaging/systemd/headscale.service

```diff
@@ -14,7 +13,7 @@ Restart=always
 RestartSec=5
 
 WorkingDirectory=/var/lib/headscale
-ReadWritePaths=/var/lib/headscale /var/run
```

Comment:
This is probably good, but would this affect the grpc socket that the cli uses?

Reply:
I tested it with these high-level steps:

* Create TLS cert/key
* Enable grpc interface
* Setup headscale on another machine via: https://headscale.net/stable/ref/remote-cli
* create + list user

Worked.

Reply:
Sounds great, please proceed